### PR TITLE
UAT feedback for aggregate user status report

### PR DESCRIPTION
**Pull request by czue**

### CODE REVIEW

**Review by esoergel — APPROVED** [review]
kicked build

File: corehq/apps/reports/templates/reports/async/aggregate_user_status.html [hunk 1]
```diff
@@ -11,12 +11,12 @@ <h4>{% trans "Users" %}</h4>
               <h2>{{ total_users }}</h2>
           </div>
           <div class="col-sm-4 text-center">
-              <h4>{% trans "Users that have Ever Submitted Data" %}</h4>
-              <h2>{{ submission_series.bucket_series.total_percent }} <small>({{ submission_series.bucket_series.total }})</small></h2>
+              <h4>{% trans "Users who have Ever Submitted Data" %}</h4>
```
Comment (proteusvacuum) [inline corehq/apps/reports/templates/reports/async/aggregate_user_status.html]: Is there a reason for this weird capitalization?

Comment (czue) [reply corehq/apps/reports/templates/reports/async/aggregate_user_status.html]: ha. "important word case" i guess. what do you think would be better?

Comment (czue) [reply corehq/apps/reports/templates/reports/async/aggregate_user_status.html]: gonna merge but happy to follow up. agree is a little funny though no one complained during UAT.

Comment (proteusvacuum) [reply corehq/apps/reports/templates/reports/async/aggregate_user_status.html]: I would have said either Title Case (Users Who Have Ever Submitted Data), or sentence case (Users who have ever submitted data). I guess we don't have convention for this type of thing, but this just struck me as weird. (non-blocking, and probably not worth changing)

**Pull request merged**